NASA-AMMOS/aerie-ui
# Activity error rollup

Resolves #936 
Bonus: Resolves #462 

To test:
1. Create a banananation plan with a few `BakeBananaBread` activities
2. Keep one untouched (this should trigger the "missing" error)
3. In another one, check on `glutenFree` and make the temperature less than 100 deg (this should trigger "invalid" errors)
4. In hasura, navigate to http://localhost:8080/console/data/AerieMerlin/schema/public/tables/activity_directive/browse
5. Edit a `BakeBananaBread` activity that aren't the ones in steps 2 and 3
6. Add `arguments` that aren't `glutenFree`, `tbSugar`, or `temperature` (this should trigger "extra" errors) 
7. Verify that errors show up in the new console area
8. Verify that errors show up at the top of the selected activity form when present
9. Verify that missing parameter errors now show up in the selected activity form
10. Verify that extra parameters are displayed in the selected activity form
11. Verify that holding `cmd/ctrl` + click on the "extra" and "invalid" error icons in the selected activity form gets rid of their corresponding errors
